install_api.sh

```diff
@@ -415,10 +415,13 @@ setup_api() {
     fi
     exec_cmd "chown -R root:ossec $API_PATH/node_modules"
     exec_cmd "chmod -R 750 $API_PATH/node_modules"
-
-    if [ ! -f $API_PATH/configuration/auth/htpasswd ]; then
-        exec_cmd "ln -s $API_PATH/node_modules/htpasswd/bin/htpasswd $API_PATH/configuration/auth/htpasswd"
+    
+    # it is necessary to remove htpasswd in order to correctly create the symbolic link
+    if [ -e $API_PATH/configuration/auth/htpasswd ]; then
```

Comment:
`if [ -e some_file ]; then...` returns true when the file `some_file` exists. This means that this file can be a regular file or a symbolic link. Reading issue #203, I guess that the API breaks when `htpasswd` becomes in a regular file instead of a symbolic link to `$API_PATH/node_modules/htpasswd/bin/htpasswd`.

So the proper check is `if [ -f some_file ]; then...`